did_x509/did_x509_test.go

```diff
@@ -21,13 +21,16 @@ func TestPercentEncode(t *testing.T) {
 		{"a+b=c", "a%2Bb%3Dc"},
 		{"~!@#$%^&*()_+", "%7E%21%40%23%24%25%5E%26%2A%28%29_%2B"},
 		{"FauxCare & Co", "FauxCare%20%26%20Co"},
+		{"FåúxCaré & Có", "F%C3%A5%C3%BAxCar%C3%A9%20%26%20C%C3%B3"},
```

Comment:
Can you check if the `url.PathUnescape` from line 119 can decode this back into the original string/